posts/Fabio/2018-04-16-hibernate-ogm-5-4-Alpha1-released.adoc

```diff
@@ -0,0 +1,68 @@
+= "Hibernate OGM 5.4.0.Alpha1 is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce the release of:
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.Alpha1]
+
+Hibernate OGM is now compatible with Hibernate ORM 5.3 and JPA 2.2!
+
+== Component upgrades
+
+ * Hibernate ORM to 5.3.0.CR1
+ * Hibernate Search to 5.10.0.Beta2
+ * WildFly to 12.0.0
+ * Infinispan to 9.2.0.Final
+ * MongoDB to 3.6.3
+ * Neo4J to 3.3.5
+
+== Module Distribution
+
+ * WildFly modules are now available as feature packs
+
+== Infinispan
+
+ * Caches created by the dialect are now transactional by default
+ * Caches created by the dialect can be configured using a cache configuration
+
+== MongoDB
+
+ * Stored procedures with positional parameters are now supported
+ * New MongoDB CLI operations supported as native queries
+
+== Release Notes
+
+All the changes are described in the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31632
+
+== Where can I get it?
+
+You can include in your project the dialect of your choice using this Maven coordinates:
+
+* http://infinispan.org[Infinispan]
+** Embedded: _org.hibernate.ogm:hibernate-ogm-infinispan-embedded:5.4.0.Alpha1_
+** Remote: _org.hibernate.ogm:hibernate-ogm-infinispan-remote:5.4.0.Alpha1_
+* https://www.mongodb.com[MongoDB]: _org.hibernate.ogm:hibernate-ogm-mongodb:5.4.0.Alpha1_
+* http://neo4j.com[Neo4j]: _org.hibernate.ogm:hibernate-ogm-neo4j:5.4.0.Alpha1_
+
+Alternatively, you can download archives containing all the binaries, source code and documentation
+https://sourceforge.net/projects/hibernate/files/hibernate-ogm/5.4.0.Alpha1[from Sourceforge].
+
+If you are interested about available versions, you can check the official
+http://hibernate.org/ogm/releases[Hibernate OGM download page].
```

Comment:
I think we should also mention the feature packs or/and add a link to the documentation